### PR TITLE
Generate all attributes in incubating artifact

### PR DESCRIPTION
**Pull request by lmolkova**
An attempt to generate everything in incubating artifact following https://github.com/open-telemetry/semantic-conventions-java/pull/45#issuecomment-1973934974

Might need a bit more polishing, but possible with https://github.com/open-telemetry/build-tools/pull/288

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -99,7 +100,8 @@ fun generateTask(taskName: String, incubating: Boolean) {
         "--file-per-group", "root_namespace",
         "-Dfilter=${filter}",
         "-DclassPrefix=${classPrefix}",
-        "-Dpkg=$packageNameArg"))
+        "-Dpkg=$packageNameArg",
+        "-DstablePkg=$stablePackageNameArg"))
```
Comment (lmolkova) [inline build.gradle.kts:104]: needed to generate `@deprecated {@link use.stable.attribute#FOO}`

File: buildscripts/templates/SemanticAttributes.java.j2 [hunk 2]
```diff
@@ -28,8 +28,8 @@
     {{type | to_camelcase(False)}}Key
   {%- endif -%}
 {%- endmacro %}
-{%- macro print_value(type, value) -%}
-  {{ "\"" if type == "String"}}{{value}}{{ "\"" if type == "String"}}
+{%- macro stable_class_ref(const_name) -%}
+{{stablePkg}}.{{file_name[:-5].replace("Incubating", "")}}#{{const_name}}
```
Comment (lmolkova) [inline buildscripts/templates/SemanticAttributes.java.j2:32]: `file_name` is  `{root_namespace}IncubatingAttributes.java`

this is ugly and relies on certain way to name files, so might need to think about better way in the future.

For now we don't really need to generate any stable attributes in the incubating artifact.

Comment (jack-berg) [reply buildscripts/templates/SemanticAttributes.java.j2:32]: > For now we don't really need to generate any stable attributes in the incubating artifact.

Are you suggesting we punt on this whole PR until some later point?

**Comment by jack-berg** [comment]
I think this is a good idea and I'm inclined to merge it. I do have some changes locally that allow it to work with the latest `otel/semconvgen:feature-codegen-by-namespace` image.

**Pull request merged**